AutoUsing/Proxy/Response.cs

```diff
@@ -0,0 +1,17 @@
+using Newtonsoft.Json;
+
+namespace AutoUsing
+{
+    /// <summary>
+    /// The JSON serialized response returned to Visual Studio Code. 
+    /// </summary>
+    public abstract class Response<T>
```

Comment:
Explain what is the generic argument (T)

Reply:
For the implicit operator to be able to serialize a JSON objects for classes that inherit Response.

Reply:
(in the xml doc)